### PR TITLE
document shutdown of http server

### PR DESCRIPTION
**Pull request by soryy708**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 2804
https://github.com/nestjs/docs.nestjs.com/issues/2804

## What is the new behavior?
Documentation about shutting down the HTTP server when starting one ourselves

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/faq/multiple-servers.md [hunk 1]
```diff
@@ -39,8 +39,41 @@ const app = await NestFactory.create(
...
-https.createServer(httpsOptions, server).listen(443);
+const httpServer = http.createServer(server).listen(3000);
+const httpsServer = https.createServer(httpsOptions, server).listen(443);
+```
+
+Because we called `http.createServer` / `https.createServer` ourselves, NestJS doesn't close them when calling `app.close` / on termination signal. We need to do this ourselves:
+
+```typescript
+@Injectable()
+export class ShutdownObserver implements OnApplicationShutdown {
```
Comment (micalevisk) [inline content/faq/multiple-servers.md:50]: can we clarify somehow where to register this `ShutdowObserver`? maybe in `AppModule`?

Comment (soryy708) [reply content/faq/multiple-servers.md:50]: Like how? Something like

```
Add `ShutdownObserver` to the `providers` of the `AppModule`
```

?

Comment (micalevisk) [reply content/faq/multiple-servers.md:50]: yeah, sounds good

**Comment by kamilmysliwiec** [comment]
LGTM, thank you 🙌

**Pull request merged**